abaplint.json

```
@@ -0,0 +1,190 @@
+{
+  "global": {
+    "files": "/src/**/*.*",
```

for `zlogger_demo_06`, I'd say dont do demos that hook into SAP standard

but you probably just want to exclude it from the checks,

```suggestion
    "files": "/src/**/*.*",
    "exclude": ["zlogger_demo_06"],
```